### PR TITLE
Dev/error and params for endpoints

### PR DESCRIPTION
**Pull request by Nordgaren**
Builders haven't been started, yet, but got the optional params started, in preparation for that.

Now only have one error to unwrap after calling a library function.

### CODE REVIEW

File: src/builders.rs [hunk 1]
```diff
@@ -0,0 +1,160 @@
+//soon
```
Comment (vswarte) [inline src/builders.rs]: Soon :tm:

File: src/api_status.rs [hunk 2]
```diff
@@ -43,11 +44,5 @@ pub mod tests {
     fn can_get_api_info() {
         let client = ShodanClient::new(get_test_api_key());
         let response = client.get_api_info().unwrap();
-
```
Comment (vswarte) [inline src/api_status.rs:46]: Why are you removing this?

**Review by vswarte — CHANGES_REQUESTED** [review]
Remove the `//soon` comment and reinstate the assert messages and we should be good

File: src/api_status.rs (hunk 2 above)
Comment (Nordgaren) [reply src/api_status.rs:46]: There is nothing to assert. The error will be known during .unwrap().

**Pull request merged**